src/common/modules/UnicodeFontHandler.js

```diff
@@ -0,0 +1,245 @@
+"use strict";
+
+import * as AddonSettings from "/common/modules/AddonSettings/AddonSettings.js";
+// import * as BrowserCommunication from "/common/modules/BrowserCommunication/BrowserCommunication.js";
+import { isMobile } from "./MobileHelper.js";
+
+import { COMMUNICATION_MESSAGE_TYPE } from "/common/modules/data/BrowserCommunicationTypes.js";
+import { fonts } from "/common/modules/data/Fonts.js";
+
+const caseIds = Object.freeze(["Lowercase", "Uppercase", "Capitalize Each Word", "Toggle Case"]);
+const fontIds = Object.freeze(["Superscript", "Small Caps", "All Small Caps", "Unicase", "separator", "Serif bold", "Serif italic", "Serif bold italic", "Sans-serif", "Sans-serif bold", "Sans-serif italic", "Sans-serif bold italic", "Script", "Script bold", "Fraktur", "Fraktur bold", "Monospace", "Double-struck", "separator", "Circled", "Circled (black)", "Squared", "Squared (black)", "Fullwidth"]);
+
+// Thunderbird
+const THUNDERBIRD = typeof messenger !== "undefined";
+
+// Chrome
+const CHROME = Object.getPrototypeOf(browser) !== Object.prototype;
+
+/**
+ * Change Unicode font.
```

Comment:
Hmm this function has a very high complexity and is not easy to understand. Can we improve that somehow?

Either with inline comments or here in the descritption it would be good to describe what/how the function does this conversation. E.g. "It replaced each char in the text with… in order to achieve the visual appearance…") Something like that.

Reply:
I will update the description.